modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js

```diff
@@ -15,14 +15,18 @@
 import {LiferayAdapt} from '../../../common/services/liferay/adapter';
 import {axios} from '../../../common/services/liferay/api';
 
```

Comment:
It might be a good practice to do it like this:
```suggestion
const headlessAPI =  'o/headless-commerce-delivery-catalog/v1.0';
```

Reply:
Done!